### DISCUSSION TITLE
YouTube has a new ad block pop-up

### DISCUSSION THREAD

**Discussion by NBspaceCadet360** [uBlockOrigin/uBlock-issues#2890] [General]
I am using no-updates option on Chrome browser for about 2 weeks.
Today I get a new ad-bloxker-blocker from Google popping into my screen.  If I turn off uBO it doesn't pop in. 
Here is a screenshot.

![yt_adblock](https://github.com/uBlockOrigin/uBlock-issues/assets/28228292/7b42da7f-a05e-4763-855e-934403878a26)

I go to watch a YouTube video, the page loads normally and when the video is just about to play this blocker screen appears.
First it gave me a count down. "You have 3 more chances to remove your ad-blocker" blah, blah....
I use Windows 11 Home  build: 22621.2428
I have Chrome -  Version 117.0.5938.134 (Official Build) (64-bit)\

**Comment by ghost** [comment]
https://github.com/uBlockOrigin/uAssets/issues/19976

**Reply by NBspaceCadet360** [reply]
I have uBO Version 1.52.2. I went through some of this before but I cannot get 'purge cache' tab to work. 
I will try everything in your link .... 
Thanks!

**Reply by gorhill** [reply]
Every single word in the instruction matters, even the parts which you might think do not. We've had cases of people not wholly disabling their other extensions because "they are not for Youtube", just to find out some interfere negatively.
